Jenkinsfile

```diff
@@ -188,12 +134,59 @@ pipeline {
                       }
                       unstash 'source'
                       dir("${BASE_DIR}"){
-                        bat """
-                        nuget restore ElasticApmAgent.sln
-                        msbuild
-                        """
+                        bat '.ci/windows/msbuild.bat'
+                      }
+                    }
+                  }
+                  /**
+                  Execute unit tests.
+                  */
+                  stage('Test') {
+                    steps {
+                      dir("${BASE_DIR}"){
+                        deleteDir()
+                      }
+                      unstash 'source'
+                      dir("${BASE_DIR}"){
+                        powershell label: 'Install test tools', script: "${readFile('./.ci/windows/test-tools.ps1')}"
+                        bat label: 'Build', script: '.ci/windows/msbuild.bat'
+                        bat label: 'Test & coverage', script: '.ci/windows/test.bat'
+                        powershell label: 'Convert Test Results to junit format', script: "${readFile('.ci/windows/convert.ps1')}"
```

Comment:
```suggestion
                        powershell label: 'Convert Test Results to junit format', script: '.\\.ci\\windows\\convert.ps1'
                       
```

Reply:
Does this apply to the `powershell` steps only? Are `bat` steps affected too?